draft-ietf-httpbis-sfbis.md

```diff
@@ -398,9 +399,9 @@ Note that the serialization algorithm ({{ser-decimal}}) rounds input with more t
 
 Strings are zero or more printable ASCII {{!RFC0020}} characters (i.e., the range %x20 to %x7E). Note that this excludes tabs, newlines, carriage returns, etc.
 
-Unicode is not directly supported in Strings, because it causes a number of interoperability issues, and -- with few exceptions -- field values do not require it.
+Non-ASCII characters are not directly supported in Strings, because it causes a number of interoperability issues, and -- with few exceptions -- field values do not require it.
```

Comment:
```suggestion
Non-ASCII characters are not directly supported in Strings, because they cause a number of interoperability issues, and -- with few exceptions -- field values do not require them.
```